nft_ingester/src/program_transformers/bubblegum/update_metadata.rs

```diff
@@ -0,0 +1,292 @@
+                    JsonValue::String("processing".to_string()),
+                    slot_i,
+                    Some(true),
+                    name.into_bytes().to_vec(),
+                    symbol.into_bytes().to_vec(),
+                )
+                .await?;
+
+                // Upsert into `asset` table.
+
+                // Set base mint info.
+                let tree_id = bundle.keys.get(5).unwrap().0.to_vec();
+                let seller_fee_basis_points =
+                    if let Some(seller_fee_basis_points) = update_args.seller_fee_basis_points {
+                        seller_fee_basis_points
+                    } else {
+                        current_metadata.seller_fee_basis_points
+                    };
+
+                unprotected_upsert_asset_base_info(
+                    txn,
+                    id_bytes.to_vec(),
+                    OwnerType::Single,
+                    false,
+                    SpecificationVersions::V1,
+                    SpecificationAssetClass::Nft,
+                    RoyaltyTargetType::Creators,
+                    None,
+                    seller_fee_basis_points as i32,
+                    slot_i,
+                )
+                .await?;
+
+                // Partial update of asset table with just compression info elements.
+                upsert_asset_with_compression_info(
+                    txn,
+                    id_bytes.to_vec(),
+                    true,
+                    false,
+                    1,
+                    None,
+                    false,
+                )
+                .await?;
+
+                // Partial update of asset table with just leaf.
+                upsert_asset_with_leaf_info(
+                    txn,
+                    id_bytes.to_vec(),
+                    nonce as i64,
+                    tree_id,
+                    le.leaf_hash.to_vec(),
+                    le.schema.data_hash(),
+                    le.schema.creator_hash(),
+                    seq as i64,
+                )
+                .await?;
+
+                // Partial update of asset table with just leaf owner and delegate.
+                let delegate = if owner == delegate || delegate.to_bytes() == [0; 32] {
+                    None
+                } else {
+                    Some(delegate.to_bytes().to_vec())
+                };
+                upsert_asset_with_owner_and_delegate_info(
+                    txn,
+                    id_bytes.to_vec(),
+                    owner.to_bytes().to_vec(),
+                    delegate,
+                    seq as i64,
+                )
+                .await?;
+
+                upsert_asset_with_seq(txn, id_bytes.to_vec(), seq as i64).await?;
+
+                // Upsert into `asset_v1_account_attachments` table.
+                let (edition_attachment_address, _) = find_master_edition_account(&id);
+                unprotected_upsert_asset_v1_account_attachments(
```

Comment:
Does account attachments do anything right now?

Assuming it does, why is it safe to do "unprotected" here?

Reply:
I don't think it does anything for compressed assets and not sure why its there as it is updated by save_v1_asset.  Maybe this is an opportunity to just remove it from compressed indexing.

The reason it was turned into an "unprotected" upsert was just noting that it was to be blanket protected by `update_metadata_seq`.

Reply:
Removed insert to `asset_v1_account_attachments` in Bubblegum parsing in https://github.com/metaplex-foundation/digital-asset-rpc-infrastructure/pull/134